recipe/build.sh

```diff
@@ -85,6 +85,7 @@ fi
             --host="${HOST}" \
             --build="${BUILD}" \
             --with-zlib="${PREFIX}" \
+            --with-szlib="${PREFIX}" \
```

Comment:
You might need to export something to help hdf5 identify the configuration for cross compilation

Reply:
Took me a bit to work out what you meant, but I eventually tracked down which bit of the configure was failing. I can't check it here so let's push and see what happens